### PR TITLE
Add topology pipeline error

### PR DESCRIPTION
**Pull request by andrewballantyne**
**Fixes**: 
https://issues.redhat.com/browse/ODC-4195

**Analysis / Root cause**: 
Continuation of #5956 - need to see the log snippet on the Topology sidebar

**Solution Description**: 
Reuses and modifies the previous log snippet code to include a design that works for both situations. Also borrows the build error block

~~**Note:** Built on top of #6003 to fix a Topology issue around rendering Pipeline Runs.~~

**Screen shots / Gifs for design review**: 
@openshift/team-devconsole-ux 
/cc @bgliwa01

Pipeline referencing a task that does not exist:
Screen Shot 2020-07-15 at 3 58 40 PM

Task that is incomplete:
Screen Shot 2020-07-15 at 3 58 33 PM

Cancelled Pipeline:
Screen Shot 2020-07-15 at 3 58 23 PM

Builds that have multiple failures:
Screen Shot 2020-07-15 at 4 01 41 PM

Pipeline Runs with multiple failures:
Screen Shot 2020-07-15 at 4 02 08 PM

Pipeline Run failure mid-execution:
Screen Shot 2020-07-15 at 3 58 12 PM

Animation of the above failure:
LogFailureAndPodLoad_topology

**Test setup:**
Same as #5956 

- Pipeline Operator installed
- Scenario "Failure at PLR validation": Any Pipeline that references a task that does not exist
- Scenario "Failure at TR validation": Create the git-clone through the Pipeline Builder (it needs a workspace and we have a support gap here)
- Scenario "Failure during run": Use these [set of pipeline resources](https://github.com/andrewballantyne/pipeline-scratch-pad/tree/master/examples/broken-app)

**Browser conformance**: 
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by jerolimov** [comment]
Impressive. Testing it locally and it works great.

@andrewballantyne: Is it possible to show a start button which opens the Pipeline Start modal when there is no pipeline run started yet?

pr6005-is-it-possible-to-show-a-start-pipeline-button-here

**Comment by andrewballantyne** [comment]
@jerolimov So we discussed this before and went to create a story to address it - after a discussion we decided we'd go with a more proactive stance and start the Pipeline out of the add flow, which is logged here: https://issues.redhat.com/browse/ODC-3779

File: frontend/public/components/overview/_build-overview.scss [hunk 1]
```diff
@@ -9,22 +9,6 @@
...
-
-.build-overview__log-snippet {
-  margin: 0;
-}
-
 .build-overview__status {
   flex-wrap: wrap;
 }
-
-.build-overview__status-message {
```
Comment (andrewballantyne) [inline frontend/public/components/overview/_build-overview.scss:28]: Forgot to clean up the styles - FYI @jerolimov

**Comment by andrewballantyne** [comment]
/assign @christianvogt 

Need a review from you Christian.

**Pull request merged**